src/kinetics/include/antioch/falloff_threebody_reaction.h

```diff
@@ -0,0 +1,249 @@
+//-----------------------------------------------------------------------bl-
+//--------------------------------------------------------------------------
+//
+// Antioch - A Gas Dynamics Thermochemistry Library
+//
+// Copyright (C) 2014 Paul T. Bauman, Benjamin S. Kirk, Sylvain Plessis,
+//                    Roy H. Stonger
+// Copyright (C) 2013 The PECOS Development Team
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the Version 2.1 GNU Lesser General
+// Public License as published by the Free Software Foundation.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc. 51 Franklin Street, Fifth Floor,
+// Boston, MA  02110-1301  USA
+//
+//-----------------------------------------------------------------------el-
+
+#ifndef ANTIOCH_FALLOFF_THREE_BODY_REACTION_H
+#define ANTIOCH_FALLOFF_THREE_BODY_REACTION_H
+
+// Antioch
+#include "antioch/antioch_asserts.h"
+#include "antioch/cmath_shims.h"
+#include "antioch/kinetics_conditions.h"
+#include "antioch/reaction.h"
+#include "antioch/lindemann_falloff.h"
+#include "antioch/troe_falloff.h"
+
+//C++
+#include <string>
+#include <vector>
+#include <iostream>
+
+namespace Antioch
+{
+  /*!\class FalloffThreeBodyReaction
+  * Base class for falloff processes coupled with efficiencies
+  *
+  * This class encapsulates a bastard between falloff and three-body reaction.
```

Comment:
I understand you don't like this model, but let's clean this up a bit. Put something like "Note: @SylvainPlessis *strongly* disapproves of the use of this model on the grounds ... <fill in your reasoning here>."